### DISCUSSION TITLE
[Guide] Step-by-step instructions

### DISCUSSION THREAD

**Discussion by zenceLde** [jheddings/notes2notion#106] [General]
Jason I'm so thankful for your script! It worked.
For me it took some help via ChatGPT, because the given resources overwhelmed me.

I want to share the Steps to Success (for me):

### Prerequisites
**1. Python3 and Poetry:** 
Ensure you have Python3 and Poetry installed. You can install them using Homebrew (https://brew.sh):
brew install python3
brew install poetry

**2. Clone the Repository:**
Download the notes2notion repository from GitHub:
git clone https://github.com/jheddings/notes2notion.git
cd notes2notion

### Install Dependencies
Navigate to the notes2notion directory (cd notes2notion) and install the dependencies: poetry install --sync

### Create a Notion Integration

**1. Create an Integration:**
- Go to Notion My Integrations: https://www.notion.so/my-integrations
- Click on 'New integration'.
- Name your integration and select the target workspace.
- Copy the 'Internal Integration Token' and save it securely. This is your NOTION_ACCESS_TOKEN.

**2. Share the Integration with a Notion Page:**
Step-by-Step Guide to Import Apple Notes into Notion Using notes2notion
- Open the Notion page you want to use.
- Click on the three dots in the top-right corner and select 'Add connections'.
- Search for your integration name and add it.

**3. Find the Page ID**
Get the Page ID:
- Open the Notion page in your browser.
- The URL should look something like:
https://www.notion.so/workspace-name/Page-Name-abcdefgh12345678.
- The page ID is the string of characters at the end of the URL, in this case, abcdefgh12345678.

### FINALLY
poetry run notes2notion --auth NOTION_ACCESS_TOKEN --page PAGE_ID

**Comment by jheddings** [comment]
Great write up!  I'm glad to hear that it is working for you.
